app/admin/projects.rb

```diff
@@ -57,6 +58,7 @@
       f.input :notes
       f.input :is_active, label: 'Active?'
       f.input :is_approved, label: 'Approved?'
+      f.input :twitter, label: 'Twitter handle'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.